packages/@aws-cdk/aws-ec2/lib/vpc-ref.ts

```diff
@@ -77,13 +77,9 @@ export interface IVpcNetwork extends IConstruct {
   subnetInternetDependencies(selection?: SubnetSelection): IDependable;
 
   /**
-   * Return whether the given subnet is one of this VPC's public subnets.
-   *
-   * The subnet must literally be one of the subnet object obtained from
-   * this VPC. A subnet that merely represents the same subnet will
-   * never return true.
+   * Return whether all of the given subnets are from the VPC's public subnets.
    */
-  isPublicSubnet(subnet: IVpcSubnet): boolean;
+  isPublicSubnets(subnetIds: string[]): boolean;
```

Comment:
not sure if idiomatic, but `arePublicSubnets` would be grammatically correct.

Reply:
Yeah, I know. But `isXxx()` is such a standard, didn't want to deviate from it.

How strongly do you feel about this?

Reply:
Not strongly at all.